__tests__/__fixtures__/output/python/requests/multipart-form-data-no-params.py

```diff
@@ -2,9 +2,8 @@
 
 url = "https://httpbin.org/anything"
 
-payload = ""
 headers = {"Content-Type": "multipart/form-data"}
 
-response = requests.request("POST", url, data=payload, headers=headers)
```

Comment:
Though we're no longer sending an empty string `data` argument to `requests` anymore for this example, what we get back from HTTPBin hasn't changed:

#### Before, with empty payload
```
> python3 __tests__/__fixtures__/output/python/requests/multipart-form-data-no-params.py
{
  "args": {}, 
  "data": "", 
  "files": {}, 
  "form": {}, 
  "headers": {
    "Accept": "*/*", 
    "Accept-Encoding": "gzip, deflate", 
    "Content-Length": "0", 
    "Content-Type": "multipart/form-data", 
    "Host": "httpbin.org", 
    "User-Agent": "python-requests/2.27.1", 
    "X-Amzn-Trace-Id": "REDACTED"
  }, 
  "json": null, 
  "method": "POST", 
  "origin": "REDACTED", 
  "url": "https://httpbin.org/anything"
}
```

#### After
```
> python3 __tests__/__fixtures__/output/python/requests/multipart-form-data-no-params.py
{
  "args": {}, 
  "data": "", 
  "files": {}, 
  "form": {}, 
  "headers": {
    "Accept": "*/*", 
    "Accept-Encoding": "gzip, deflate", 
    "Content-Length": "0", 
    "Content-Type": "multipart/form-data", 
    "Host": "httpbin.org", 
    "User-Agent": "python-requests/2.27.1", 
    "X-Amzn-Trace-Id": "REDACTDD"
  }, 
  "json": null, 
  "method": "POST", 
  "origin": "REDACTED", 
  "url": "https://httpbin.org/anything"
}
```